runtime/onert/core/src/backend/BackendContext.cc

```diff
@@ -56,9 +57,7 @@ void BackendContext::initConsts()
   {
     const auto &obj = _graph->operands().at(ind);
     if (obj.isConstant() && !constant_initializer->exist(ind))
-    {
-      constant_initializer->registerPermuteInitializer(ind, obj);
-    }
+      constant_initializer->registerDefaultInitializer(ind, obj);
```

Comment:
@wateret how about this? According to your comemnt, I revise with `registerDefaultInitializer`.

Reply:
Looks much better 😄